controllers/nginx/pkg/cmd/controller/nginx.go

```diff
@@ -252,6 +253,24 @@ func (n NGINXController) OnUpdate(cmap *api.ConfigMap, ingressCfg ingress.Config
 	}, n.testTemplate)
 }
 
+// Name returns the healthcheck name
+func (n NGINXController) Name() string {
+	return "Ingress Controller"
+}
+
+// Check returns if the nginx healthz endpoint is returning ok (status code 200)
+func (n NGINXController) Check(_ *http.Request) error {
+	res, err := http.Get("http://127.0.0.1:18080/healthz")
```

Comment:
Yes, the port if 10254 (can be changed, is behind a flag) and all the examples use this port for liveness and readiness check
https://github.com/kubernetes/ingress/blob/master/core/pkg/ingress/controller/launch.go#L67
https://github.com/kubernetes/ingress/pull/31/files#diff-ce130ef9249cc52ef757bf889e4ba07dR160